drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -22,6 +22,11 @@ matterManufacturer:
     deviceProfileName: plug-binary
 
 matterGeneric:
+  - id: "matter/bridge"
```

Comment:
I put it first in this list because I think that has an affect on tie breaks where it matches other device types. @tpmanley do you think I should add a couple more generic fingerprints that also include On/Off and Dimmer device types? This might help ensure the bridges get the right fingerprint.

Reply:
We determined that the number of permutations needed for this to be effective would be untenable. I am leaving the generic fingerprint in for when official bridge support is released to hubs, and existing bridge devices will be identified with manufactuer specific fingerprints.